brownie/__init__.py

```diff
@@ -19,7 +19,7 @@
 __all__ = [
     'accounts',
     'alert',
-    'history',
+    'state',
```

Comment:
This is a great example of why going with option 2 instead of 3 is a better choice.  In this case, you do still want to import the instantiated version of `TxHistory` as `history`.  If you reverse this change, that last test should be passing again.